components/SearchResult.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+    <div class="flex flex-col">
```

Comment:
This `flex flex-col` class shouldn't be needed since that's the default orientation. If this is removed then all of the `flex flex-row` can just be changed to `flex`.

Reply:
nice catch! updated

Reply:
honestly, I'm used to being overexplicit on flex classes. 
Good for me to trim out defaults more often 👍